keyboards/7skb/rev1/rev1.c

```diff
@@ -0,0 +1,2 @@
+#include "7skb.h"
```

Comment:
This should actually be

```suggestion
#include "rev1.h"
```